src/main/java/eu/dissco/orchestration/backend/controller/MachineAnnotationServiceController.java

```diff
@@ -63,9 +64,9 @@ public ResponseEntity<JsonApiWrapper> updateMachineAnnotationService(
       Authentication authentication,
```

Comment:
[Line 58] is the method getUserId() used? my IDE says no.

Reply:
Good catch, changed it around and the method lost its purpose